lib/rules/space-infix-ops.js

```diff
@@ -51,7 +53,7 @@ module.exports = {
 
         // Build maps of operators that must or must not
         // have spaces around them.
-        const spacesYes = {};
+        const spacesYes = {in: true, instanceof: true};
```

Comment:
You'll need to rebase this (and add spaces inside the braces). We just enabled `object-curly-spacing` in the ESLint codebase.